tools/ramclustr/ramclustr_define_experiment.xml

```diff
@@ -0,0 +1,198 @@
+<tool id="ramclustr_define_experiment" name="RAMClustR define experiment" version="1.0.0+galaxy0">
```

Comment:
```suggestion
<tool id="ramclustr_define_experiment" name="RAMClustR define experiment" version="1.0.0">
```

Reply:
Since this is a "galaxy only" tool, we drop the two-part versioning? Makes sense!

Reply:
Indeed, there is no other dependency that we need to harmonize with, this change makes it clear.

Reply:
Yes, makes sense, thank you!